fem/gridfunc.hpp

```diff
@@ -103,7 +103,7 @@ class GridFunction : public Vector
    /** The content of @a input should be in the format created by the method
        Save(). The reconstructed FiniteElementSpace and FiniteElementCollection
        are owned by the GridFunction. */
-   GridFunction(Mesh *m, std::istream &input);
+   GridFunction(Mesh *m, std::istream &input, bool connect = false);
```

Comment:
I must say this type of change to a very commonly used class makes me very nervous. What exactly is the purpose of this new argument? At the very least we need to document this argument here and in the other constructors with new arguments. Beyond that I wonder what impact this might have in a non-NURBS context. Even if that impact is purely psychological i.e. will users expect this argument to activate some new feature in non-NURBS settings?

Reply:
I had a look at `FormLinearSystem` and its probably a bit more involved than I anticipated...